### ISSUE TITLE
[📖] How to access the current url / path from the root context

### ISSUE THREAD

**Issue by srapport** [QwikDev/qwik#6597]
### Suggestion

It is unclear how to access the `url` / path from `root.tsx` which is sometimes necessary. There is one reference in Discord w/o resolution. This might not be big enough for a cookbook recipe, but we should have an example in the docs.

**Comment by srapport** [comment]
Per @shairez:

`const urlEnv = useServerData<string>('url');`

Where-o-where should this go in the docs... near `useLocation()` would make sense. 🤔

Turning it into a url:

```ts
const serverDataUrl = useServerData<string>('url');
const url = new URL(serverDataUrl || 'https://unkonwn');
```

**Comment by srapport** [comment]
https://github.com/QwikDev/qwik/pull/6598

**Comment by shairez** [comment]
fixed in #6598
